lib/fluent/plugin/out_elasticsearch.rb

```diff
@@ -47,7 +51,8 @@ def start
 
   def client
     @_es ||= begin
-      adapter_conf = lambda {|f| f.adapter :excon }
+      excon_options = { client_key: @client_key, client_cert: @client_cert, client_key_pass: @client_key_pass }
+      adapter_conf = lambda {|f| f.adapter :excon, excon_options }
```

Comment:
Since you're configuring Faraday's ssl options bellow (in the `transport_options` hash), do we need to configure Excon here as well?